Command/RunCommand.php

```diff
@@ -375,13 +381,24 @@ private function startJob(Job $job)
      *
      * In such an error condition, these jobs are cleaned-up on restart of this command.
      */
-    private function cleanUpStaleJobs($workerName)
+    private function cleanUpStaleJobs($workerName, $restrictedQueues)
```

Comment:
Yes, ``workerName`` can only be null when running an older version of this bundle, and then migrating to the newer version.

If you use a pre-determined worker-name, stale jobs will be detected faster, i.e. as soon as the Run command is re-started. If you use the default, the CleanUp command will eventually collect stale jobs, but that will be a bit slower (a few minutes).